cmd/tellor/main.go

```diff
@@ -44,17 +46,19 @@ func buildContext() error {
 		// Create an rpc client
 		client, err := rpc.NewClient(cfg.NodeURL)
 		if err != nil {
-			log.Fatal(err)
+			return fmt.Errorf("Couldn't create client instance: %s", err)
```

Comment:
here and few other places, lets use errors.Wrap, all lower case, and no need to use words like - error, couldn't , problem etc. these are obvious from the error log.
```suggestion
			return errors.Wrap(err,"create client instance")
```